src/app/helpers/notify.ts

```diff
@@ -12,6 +12,16 @@ function isPageVisible(): boolean {
 
 type NotificationCategory = 'Error' | 'Success' | 'Travel' | 'LocationClaim';
 
+export type EnabledCategories = Exclude<NotificationCategory, 'Error' | 'Success'>;
```

Comment:
this type should be singular, that is to say, "ToggleableCategory" or some such.